src/routes/expenses.route.js

```diff
@@ -0,0 +1,17 @@
+const expensesController = require('../controller/expenses.controller');
+const express = require('express');
+const router = express.Router();
+
+router.get('/', expensesController.get);
+
+router.get('/:id', expensesController.getOne);
+
+router.post('/', expensesController.create);
+
+router.delete('/:id', expensesController.remove);
+
+router.patch('/:id', express.json(), expensesController.update);
```

Comment:
It is a common practice to use express.json() middleware globally for all routes. Consider moving express.json() to your main server file where you initialize your express app. This way, you won't need to specify it for individual routes.